yearn/prices/magic.py

```diff
@@ -20,133 +20,44 @@
 
 logger = logging.getLogger(__name__)
 
-
+NETWORKS = [
+    Network.Arbitrum,
+    Network.Fantom,
+    Network.Mainnet
+]
 
 @ttl_cache(10000)
 def get_price(token, block=None):
-    token = str(token)
-    logger.debug("unwrapping %s", token)
-
-    if token in constants.stablecoins:
-        logger.debug("stablecoin -> %s", 1)
-        return 1
-
-    if chain.id == Network.Mainnet:
-        return get_price_eth(token, block)
-    elif chain.id == Network.Fantom:
-        return get_price_ftm(token, block)
-    elif chain.id == Network.Arbitrum:
-        return get_price_arbi(token, block)
-    else:
-        raise UnsupportedNetwork('magic price oracle is not supported on this network')
-
-
-def get_price_arbi(token, block=None):
-    price = None
-
-    if token in compound:
-        price = compound.get_price(token, block=block)
-        logger.debug("compound -> %s", price)
-    
-    if isinstance(price, list):
-        price, underlying = price
-        logger.debug("peel %s %s", price, underlying)
-        return price * get_price(underlying, block=block)
+    if chain.id not in NETWORKS:
```

Comment:
This check was there in a similar way before so I kept it. I thought it would be nice to have an explicit exception if someone started this with another unsupported chain_id.

Removed it for now, because we're injecting the environment and it will fail in many other places if it's not configured properly.